demo/react/doc/product/components/editable-media.mdx

```diff
@@ -4,13 +4,17 @@ import { Alignment, EditableMedia, Grid, GridItem, IconButton, Theme, Size } fro
 import { mdiDelete, mdiEye, mdiPencil } from 'LumX/icons';
 ```
 
-# EditableMedia
+# Editable Media
 
-**Featured Image Pick or upload a picture. It will show up in the preview of this content.**
+**Help users select or edit a media. display an image with related info. **
 
-## Default
+## Variants
 
-```javascript jsx withThemeSwitcher disableGrid
+**Editable medias come in two variants: thumbnail and avatar.**
```

Comment:
`Editable medias come in two variants: *thumbnail* and *avatar*.`

(tout est en bold actuellement, ça fait bizarre)